js/src/main/scala/com/raquo/domtypes/jsdom/defs/package.scala

```diff
@@ -2,14 +2,25 @@ package com.raquo.domtypes.jsdom
 
 import com.raquo.domtypes.generic
 import org.scalajs.dom
+import org.scalajs.dom.html
+import scala.scalajs.js.annotation.ScalaJSDefined
 
 package object defs {
 
+  @ScalaJSDefined
+  class ValueElementEvent extends dom.Event {
+    def targetValue: String = target match {
+      case input: html.Input => input.value
+      case textarea: html.TextArea => textarea.value
+      case select: html.Select => select.value
+      case elem: html.Element => elem.textContent // for contenteditable elements
```

Comment:
Hmmm doesn't look like this should work, have you tried this pattern match in practice with a contentEditable element?

`html.Element` extends `js.Any` (via a few layers), and Scala.js docs [say](https://www.scala-js.org/doc/interoperability/facade-types.html):

> `isInstanceOf[T]` is not supported for any trait `T` inheriting from `js.Any`. Consequently, pattern matching for such types is not supported either.

Reply:
Actually wait... `HTMLElement` is an abstract class, not a trait, so I guess it should work